scripts/mongo/common.inc.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+foreach (array(__DIR__ . '/../../../../autoload.php', __DIR__ . '/../../vendor/autoload.php') as $file) {
+    if (file_exists($file)) {
+        define('TRIPOD_COMPOSER_INSTALL', $file);
+        break;
+    }
+}
+
+if (!defined('TRIPOD_COMPOSER_INSTALL')) {
+    die(
+        'You need to set up the project dependencies using the following commands:' . PHP_EOL .
+        'wget http://getcomposer.org/composer.phar' . PHP_EOL .
```

Comment:
Well, we definitely shouldn't mention ant init, because the point of this is that there was an assumption that composer would be run in the tripod directory, which I think is a bad assumption.  This is intended to allow tripod to either be the "main" composer project or a dependency (i.e. has a vendor subdirectory or is in the vendor subdirectory, but not both).

I'll update the die message.